crates/astria-conductor/src/sequencer/mod.rs

```diff
@@ -377,7 +389,10 @@ fn schedule_for_forwarding_or_resubscribe(
                     "block received from sequencer subscription was bad; dropping it"
                 );
             }
-            Ok(block) => pending_blocks.push_back(futures::future::ready(block)),
+            Ok(block) => {
+                info!(at_height = %block.header().height, "received new block from sequencer subscription");
```

Comment:
The `from sequencer subscription` is redundant because this information is derived from the environment